feathr_project/feathr/client.py

```diff
@@ -415,7 +415,6 @@ def get_offline_features(self,
                              output_path: Union[str, Sink],
                              execution_configurations: Union[SparkExecutionConfiguration ,Dict[str,str]] = {},
                              config_file_name:str = "feature_join_conf/feature_join.conf",
-                             udf_files = None,
```

Comment:
Any specific reason to remove this param from `client.py`?

Reply:
The variable `udf_files` is assigned in the body of the function which will override the args anyway:
```
        udf_files = _PreprocessingPyudfManager.prepare_pyspark_udf_files(feature_names, self.local_workspace_dir)
```